components/nimbus/uniffi.toml

```diff
@@ -0,0 +1,2 @@
+[bindings.kotlin]
+package_name = "org.mozilla.experiments.nimbus"
```

Comment:
Please add this, in line with other `uniffi.toml` files in this repo:

```
cdylib_name = "megazord"
```

This will cause the bindings to look for the library name in the correct `.so` by default, removing the need for [this libraryOverride setting](https://github.com/mozilla-mobile/android-components/blob/0bc7edf4a84a437c3819bd11f212d47998ba02e3/components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt#L240-L245) inside of android-components.

We should also consider whether we want to change the official kotlin package name from `org.mozilla.experiments.nimbus` to `org.mozilla.appservices.nimbus` for consistency. I don't feel too strongly on that front, but also, I think it would be a pretty minor change for the android component. /cc @travis79.

Reply:
Done, and filed mozilla-mobile/android-components#9969 for the removal